shared/cfg/guest-os/Linux/RHEL/8/8.0.cfg

```diff
@@ -0,0 +1,9 @@
+- 0:
```

Comment:
as mentioned in the main chat, this should not be here as 8.0 is not yet released and then it'll require the DVD/kernel/initrd hashes specified.